src/screens/Portfolio/index.js

```diff
@@ -198,28 +150,29 @@ export default function PortfolioScreen({ navigation }: Props) {
             : []),
           ListHeaderComponent(),
           StickyActions(),
-          <SectionList
-            // $FlowFixMe
-            sections={sections}
-            style={styles.list}
-            contentContainerStyle={styles.contentContainer}
-            keyExtractor={keyExtractor}
-            renderItem={renderItem}
-            // $FlowFixMe
-            renderSectionHeader={renderSectionHeader}
-            onEndReached={onEndReached}
-            stickySectionHeadersEnabled={false}
-            ListFooterComponent={
-              !completed ? (
-                <LoadingFooter />
-              ) : accounts.every(isAccountEmpty) ? null : sections.length ? (
-                <NoMoreOperationFooter />
-              ) : (
-                <NoOperationFooter />
-              )
-            }
-            ListEmptyComponent={ListEmptyComponent}
-          />,
+          ...(showDistribution
+            ? [
```

Comment:
Interesting, what does this syntax do? Why not `showDistribution ? <View style={styles.distrib}> etc...`

Reply:
we are creating an array of components to show in a list here so thats the syntax :sweat_smile: i should move it outside of the template though thanks